exporter/tanzuobservabilityexporter/metrics.go

```diff
@@ -188,6 +190,31 @@ func getValue(numberDataPoint pdata.NumberDataPoint) (float64, error) {
 	}
 }
 
+// pushGaugeSingleNumberDataPoint sends a metric as a gauge metric to tanzu
+// observability. metric is the metric to send. numberDataPoint is the value
+// of the metric. Any errors get appended to errs. sender is what sends the
+// gauge metric to tanzu observability. logger is the logger. missingValues
+// keeps track of metrics with missing values.
+func pushGaugeSingleNumberDataPoint(
+	metric pdata.Metric,
+	numberDataPoint pdata.NumberDataPoint,
+	errs *[]error,
+	sender gaugeSender,
+	logger *zap.Logger,
+	missingValues *counter) {
```

Comment:
To make it easier to read where parameters end and the function body starts, please put `) {` on a newline. Same goes for any other methods in this PR that have the same style.

Reply:
Done.